examples/lib/stories/input/input.dart

```diff
@@ -97,7 +97,7 @@ void addInputStories(Dashbook dashbook) {
     ..add(
       'Gesture Hitboxes',
       (_) => GameWidget(game: GestureHitboxesExample()),
-      codeLink: baseLink('collision_detection/simple_shapes_example.dart'),
+      codeLink: baseLink('input/simple_shapes_example.dart'),
```

Comment:
This is wrong, there is no `simple_shapes_example` in `input`.
You have to verify that all links are correct, otherwise this PR doesn't help much.
You can do this by concatenating "https://github.com/flame-engine/flame/blob/main/examples/lib/stories/" with whatever you write here, and if it is wrong you can find the right one by browsing around on that base link.